sdk/purview/purview-workflow-rest/samples-dev/approveApprovalTaskSample.ts

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+import createPurviewWorkflowClient, {
+  ApproveApprovalTaskParameters,
+} from "@azure-rest/purview-workflow";
+import { UsernamePasswordCredential } from "@azure/identity";
+import * as dotenv from "dotenv";
+
+dotenv.config();
+
+/**
+ * This sample demonstrates how to Approve an approval task.
+ *
+ * @summary Approve an approval task.
+ * x-ms-original-file: specification/purview/data-plane/Azure.Analytics.Purview.Workflow/preview/2022-05-01-preview/examples/ApproveApprovalTask.json
+ */
+const endpoint = process.env["ENDPOINT"] || "";
+const tenantId = process.env["TENANTID"] || "";
+const clientId = process.env["CLIENTID"] || "";
+const username = process.env["USERNAME"] || "";
+const password = process.env["PASSWORD"] || "";
+
+async function approvalTaskApprove() {
+  const credential = new UsernamePasswordCredential(tenantId, clientId, username, password);
+  const client = createPurviewWorkflowClient(endpoint, credential);
+  const taskId = "98d98e2c-23fa-4157-a3f8-ff8ce5cc095c";
+  const options: ApproveApprovalTaskParameters = {
+    body: { comment: "Thanks for raising this!" },
+  };
+  const result = await client
+    .path("/workflowtasks/{taskId}/approve-approval", taskId)
+    .post(options);
+  console.log(result);
```

Comment:
I see that you currently have one sample file per operation. Instead I would recommend that you create sample files based on User Scenarios. For example if you expect your user to 

1. Create a workflow
2. Reassign a workflow
3. Approve a workflow

I would like to see samples that do a combination of things so that our users can get a better idea of how to use the library. There is very little value in just showing them how to call individual operations, as intellisense will tell them already.

Reply:
This applies to all sample files, I would encourage you to re-organize the samples. I would prefer to have one or two samples that show a real user scenario than many samples that just show how to make an operation call

Reply:
That make sense, fixed, thanks. Workflow is a straightforward service, there have no much complicated user scenrios, so I also added some comments in samples to explain the procedures.